app/controllers/annotation_controller.js

```diff
@@ -0,0 +1,168 @@
+import Annotation from '../models/annotation';
+
+export const createAnnotation = (req, res) => {
+  if (req.isAuthenticated()) {
```

Comment:
Oh, I see. If we are allowing unauthenticated requests in some cases, then I think it's fine how you have it now.